gdk/commands/test/config/InitConfiguration.py

```diff
@@ -17,6 +18,15 @@ def _get_gtf_version(self):
             _version_arg = self._args.get("otf_version", None)
         if _version_arg:
             logging.info("Using the GTF version provided in the command %s", _version_arg)
+            if not self.test_config.upgrade_suggestion:
+                try:
+                    if Version(_version_arg) < Version(self.test_config.latest_gtf_version):
+                        logging.info(
+                            f"The current latest version of GTF is {self.test_config.latest_gtf_version}. Please consider "
+                            "using the latest version."
+                        )
+                except Exception as e:
+                    logging.debug("Not providing GTF update suggestion due to caught version error: %s", str(e))
```

Comment:
Nit: I think we can move this `try-except` under this `_validated_gtf_version()` method and remove the `upgrade_suggestion` boolean completely.

Reply:
We will still need the boolean as the _validated_gtf_version function is called whether or not a command line arg was passed. However, I think it makes more logical sense to include the logic under that function and it is much more readable, and we can avoid the try-except logic as we'll already be validating the version beforehand. I'll move the logic over in the next commit and also renamed the boolean to be more clear what false/true means at a glance.